module_cheatsheets/scRNA-seq_cheatsheet.md

```diff
@@ -0,0 +1,81 @@
+| `ggplot2`            | [`ggtitle()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)                              | GG Title        | Sets the title of a `ggplot`                         |                                                       
+
+<div style="page-break-after: always;"></div>
+
+### `scran`, `scater`, `SingleCellExperiment`
+Read the `scran` package documentation [**here**](https://www.rdocumentation.org/packages/scran/versions/1.10.2), and a vignette on its usage [**here**](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html).
+
+Read the `scater` package documentation [**here**](https://www.rdocumentation.org/packages/scater/versions/1.10.1), and a vignette on its usage [**here**](https://github.com/davismcc/scater/blob/master/vignettes/vignette-intro.Rmd).
+
+Read the `SingleCellExperiment` package documentation [**here**](https://osca.bioconductor.org/), and a vignette on its usage [**here**](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html).
+
+<br>
+
+| Library/Package      | Piece of Code              | What it's called                           | What it does                                                                                                           |
+|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
+| `scran`                | [`quickCluster()`](https://www.rdocumentation.org/packages/scran/versions/1.0.3/topics/Quick%20clustering)                | Quick Clustering     | Groups similar cells into clusters and normalizes these cells separately. The size factors are rescaled for comparison across clusters.                               |
+| `scran`                | [`computeSumFactors()`](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html)                 | Compute Sum Factors        | Returns a numeric vector of computed sum factors for each cell cluster in the object. The cluster-based size factors are deconvolved into cell-based size factors for the normalization of each cell's gene expression profile. This is done to avoid errors that may result from bulk normalization methods.                   |
+| `scater`               | [`normalize()`](https://www.rdocumentation.org/packages/scater/versions/1.0.4/topics/normalize)                       | Normalize      | Returns the SingleCellExperiment object with normalized expression values for each cell, using the appropriate size factor and log transformation  |
+| `SingleCellExperiment`               | [`logcounts()`](https://bioconductor.org/packages/devel/bioc/vignettes/scater/inst/doc/vignette-intro.html)           | Logcounts      | Stores or extracts log transformed single cell experiment count data as an assay of the SingleCellExperiment object       |
```

Comment:
Also, the general format we've decided in the notebooks, based on how pubs use the terms, is when single-cell is used as an adjective we hyphenate, but when we are talking about single cell as in an adjective and noun, then no hyphen.

Reply:
The Amezquita would be a good addition. Since it addresses the `SingleCellExperiment` object, should we put the link in the header or would it serve as a better explanation specifically for the `logcounts` and `rowData` functions?

Reply:
I would think in the header just because it has a lot of great information in there.